### PR TITLE
Rep status update dev

### PR DESCRIPTION
**Pull request by jynCoder**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/UpdateRepresentativeStatusServlet.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+The RepInDatastore Servlet class checks to see whether or not a particular
+representative has made an account by querying for their name in datastore
+*/
+@WebServlet ("/update_representative_status")
+public class UpdateRepresentativeStatusServlet extends HttpServlet {
+    private static final Logger logger = LogManager.getLogger("UpdateRepresentativeStatusServlet");
+    private static final String REP_NAME = "repName";
+    private static final String REP_STATUS = "status";
+    
+
```
Comment (swetha-gangu) [inline src/main/java/com/google/sps/servlets/UpdateRepresentativeStatusServlet.java:25]: extra blank lines

File: src/main/java/com/google/sps/data/DatastoreManager.java [hunk 2]
```diff
@@ -244,6 +253,8 @@ public static void updatePostWithAnswer(long postId, long answerId)
         datastore.put(postEntity);
     }
 
+
```
Comment (swetha-gangu) [inline src/main/java/com/google/sps/data/DatastoreManager.java:256]: extra blank line s

File: src/main/webapp/stylesheets/style.css [hunk 3]
```diff
@@ -102,3 +102,21 @@ i {
...
+#offline{
+    color: tomato;
+}
+
+.onlineClicked{
+    background-color: lightgreen;
+    color: white;
+}
+
+offlineClicked{
```
Comment (jindaln) [inline src/main/webapp/stylesheets/style.css:119]: is this supposed to be a class? .offlineClicked {

File: src/main/webapp/stylesheets/style.css [hunk 4]
```diff
@@ -102,3 +102,21 @@ i {
 .underNavBar {
     margin-top:80px;
 }
+
+#online{
```
Comment (jindaln) [inline src/main/webapp/stylesheets/style.css:106]: Please add space before { for all of these

File: src/main/webapp/script.js [hunk 5]
```diff
@@ -812,3 +812,49 @@ function zipcodelink() {
...
+function setStatus(status){
+     if(status == "false") {
+        var online = document.getElementById("online");
+        var offline = document.getElementById("offline");
+        offline.style.backgroundColor = "tomato";
+        offline.style.color = "white";
+        online.style.backgroundColor = "white";
+        online.style.color = "lightgreen";
+    }
+    else{
```
Comment (jindaln) [inline src/main/webapp/script.js:852]: add space

File: src/main/webapp/script.js [hunk 6]
```diff
@@ -812,3 +812,49 @@ function zipcodelink() {
     var zipcode = document.getElementById("backToRepZipcode");
     zipcode.style.display = isRep.trim() == "true" ? "block" : "none"; 
 }
+
+function closeWindow(){
```
Comment (jindaln) [inline src/main/webapp/script.js:816]: add space before { for all of these

File: src/main/java/com/google/sps/servlets/UpdateRepresentativeStatusServlet.java [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+/** 
+The RepInDatastore Servlet class checks to see whether or not a particular
+representative has made an account by querying for their name in datastore
+*/
+@WebServlet ("/update_representative_status")
+public class UpdateRepresentativeStatusServlet extends HttpServlet {
+    private static final Logger logger = LogManager.getLogger("UpdateRepresentativeStatusServlet");
+    private static final String REP_NAME = "repName";
+    private static final String REP_STATUS = "status";
+
```
Comment (jindaln) [inline src/main/java/com/google/sps/servlets/UpdateRepresentativeStatusServlet.java:24]: remove extra space

File: src/main/java/com/google/sps/servlets/UpdateRepresentativeStatusServlet.java [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/** 
+The RepInDatastore Servlet class checks to see whether or not a particular
```
Comment (jindaln) [inline src/main/java/com/google/sps/servlets/UpdateRepresentativeStatusServlet.java:16]: Add appropriate Java Doc

File: src/main/java/com/google/sps/data/Representative.java [hunk 9]
```diff
@@ -70,6 +72,10 @@ public long getID() {
         return id; 
     }
 
+    public String getStatus(){
```
Comment (jindaln) [inline src/main/java/com/google/sps/data/Representative.java:75]: space before {

**Pull request merged**